### PR TITLE
[jdbc_ycsb] Support YCSB JDBC benchmark

### PR DESCRIPTION
**Pull request by k1xme**
Add YCSB benchmark against JDBC-supported databases.

This benchmark takes care of creating and dropping test table.

Have to specify the path to jdbc driver jar to run the benchmark.

### CODE REVIEW

File: perfkitbenchmarker/linux_benchmarks/jdbc_ycsb_benchmark.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
```
Comment (hildrum) [inline perfkitbenchmarker/linux_benchmarks/jdbc_ycsb_benchmark.py]: 2016

File: perfkitbenchmarker/linux_benchmarks/jdbc_ycsb_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                    "(YCSB_KEY VARCHAR(255) PRIMARY KEY, "
+                    "FIELD0 TEXT, FIELD1 TEXT, "
+                    "FIELD2 TEXT, FIELD3 TEXT, "
+                    "FIELD4 TEXT, FIELD5 TEXT, "
+                    "FIELD6 TEXT, FIELD7 TEXT, "
+                    "FIELD8 TEXT, FIELD9 TEXT);")
+DROP_TABLE_SQL = "DROP TABLE IF EXISTS usertable;"
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('db_driver',
```
Comment (hildrum) [inline perfkitbenchmarker/linux_benchmarks/jdbc_ycsb_benchmark.py]: These flags should be prefixed with the benchmark name.

**Comment by hildrum** [comment]
LGTM

**Comment by k1xme** [comment]
Ready to merge

**Pull request merged**